.github/workflows/build.yml

```diff
@@ -18,5 +18,3 @@ jobs:
         go-version: 1.15
     - name: Build
       run: make -j dist
-    - name: List output
-      run: ls -la build
```

Comment:
Why this change?

Reply:
I can revert it, but didnt see what value it provide